app/dashboard/createDashboardPageContext.test.js

```diff
@@ -374,7 +376,73 @@ describe('createDashboardPageContext', () => {
       required: ['some-first-sub-section-id'],
     };
 
-    const context = createDashboardPageContext('some-solution-id', dashboardManifest, marketingDataSections, validationErrors);
+    const context = createDashboardPageContext({
+      solutionId: 'some-solution-id', dashboardManifest, marketingDataSections, validationErrors,
+    });
+
+    expect(context).toEqual(expectedContext);
+  });
+
+  it('should create a context with the url of the sub section containing the dashboard id', () => {
+    const expectedContext = {
```

Comment:
is there a test for the url if there is no dashboardId?

Reply:
It's kinda covered in the above tests here. But there is no test explicitly to check that. Do you think it's worth adding one?

Reply:
I feel like I should always say yes to this question?!

Reply:
haha I'll get on it